cmd/build/v2/build.go

```diff
@@ -191,7 +191,7 @@ func (bc *OktetoBuilder) Build(ctx context.Context, options *types.BuildOptions)
 			}
 
 			if !okteto.Context().IsOkteto && buildSvcInfo.Image == "" {
-				return fmt.Errorf("'build.%s.image' is required if your cluster doesn't have Okteto installed", svcToBuild)
+				return fmt.Errorf("'build.%s.image' is required if your context doesn't have Okteto installed", svcToBuild)
```

Comment:
```suggestion
				return fmt.Errorf("'build.%s.image' is required if your context does not belong to Okteto.", svcToBuild)
```
I think the term `install` was focused on the `cluster`. Now that we talk about `context` I think it is more appropriate in this way.